src/FakeItEasy/MethodInfoExtensions.cs

```diff
@@ -171,9 +171,14 @@ private static void AppendParameters(StringBuilder builder, ParameterInfo[] para
             {
                 AppendParameterSeparator(builder, i);
                 builder
-                    .Append(parameters[i].ParameterType)
-                    .Append(' ')
-                    .Append(parameters[i].Name);
+                    .Append(parameters[i].ParameterType);
+                var parameterName = parameters[i].Name;
+                if (parameterName is not null)
+                {
+                    builder
+                       .Append(' ')
+                       .Append(parameterName);
+                }

```

Comment:
Based on the title and description of the PR, I didn't expect any change to the library. Maybe rename the PR to match what it actually does?

Reply:
Fair point! Updated.